src/Processors/Formats/Impl/AvroRowInputFormat.cpp

```diff
@@ -68,35 +68,35 @@ namespace ErrorCodes
     extern const int CANNOT_READ_ALL_DATA;
 }
 
-class InputStreamReadBufferAdapter : public avro::InputStream
+bool AvroInputStreamReadBufferAdapter::next(const uint8_t ** data, size_t * len)
```

Comment:
what was the reason for these changes in Avro? May be it can be submitted in a separate PR?

Reply:
I used some method here to parse the avro format iceberg meta file.

Reply:
Ok